packages/foam-core/src/core.ts

```diff
@@ -0,0 +1,89 @@
+import { Graph, Edge } from 'graphlib'
+
+
+type ID = string
+
+export interface Link {
+  from: ID
+  to: ID
+  text: string
+}
+
+export interface NoteLink {
+  to: ID
+  text: string
+}
+
+
+export class Note {
+  public id: ID
+  public title: string
+  public source: string
+  public path: string
```

Comment:
here id vs path

Reply:
Mentioned this on previous comment, but the fact that the ID happens to be a path is an implementation detail and we shouldn't rely on the fact. If we do need a file path (and we do), I think it would be good to keep it in `path`